DFC.Digital/DFC.Digital.Data/Interfaces/IWebAppContext.cs

```diff
@@ -24,7 +24,7 @@ public interface IWebAppContext
 
         bool IsUserAdministrator { get; }
 
-        NameValueCollection RequestQueryString { get; }
+        NameValueCollection RequestQueryString { get; set; }
```

Comment:
Is there a reason why we had to make it settable.
```suggestion
        NameValueCollection RequestQueryString { get; }
```